README.md

```diff
@@ -144,6 +144,7 @@ the Design of Network-based Software Architectures](https://www.ics.uci.edu/~fie
 * [Python Eve](http://python-eve.org/) - Eve is an open source Python REST API framework designed for human beings. It allows to effortlessly build and deploy highly customizable, fully featured RESTful Web Services.
 * [Ramses](https://ramses.readthedocs.org/en/stable/) - Makes RAML files executable by generating production-ready APIs from them at runtime.
 * [Flask-Potion](https://github.com/biosustain/potion) - Flask-Potion is a powerful Flask extension for building RESTful JSON APIs. It also provides several Clients for easier access to the API.
+* [admin interface for aiohttp application](https://github.com/aio-libs/aiohttp_admin) - aiohttp_admin will help you on building an admin interface on top of an existing data model. Library designed to be database agnostic and decoupled of any ORM or datbase layer. Admin module relies on async/await syntax (PEP492) thus not compatible with Python older then 3.5. It uses ng-admin as front-end layer.
```

Comment:
Looks quite long compared to other descriptions. :)

Can we perhaps simplify by the following?

> [aiohttp Admin Interface](https://github.com/aio-libs/aiohttp_admin) - Build an ng-admin powered admin interface on top of an existing aiohttp data model. Due to `async await` syntax, it requires Python >= 3.5.